test/functional/abc-block-sigchecks-activation.py

```diff
@@ -0,0 +1,355 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Developers
```

Comment:
this test was cherry picked but is in the wrong folder for BU and also we need to add it to rpc-tests.py

Reply:
I cherry picked it with no changes for record keeping but then moved and reworked it in a later commit.